config/default.json

```diff
@@ -19,5 +19,291 @@
   },
   "EMAIL_USER": "modusboxemailtest@gmail.com",
   "EMAIL_PASSWORD": "April2o0%",
-  "EMAIL_SMTP": "smtp.gmail.com"
```

Comment:
Note that the "SIDECAR.DISABLED" = **true** must be changed back to **false**

Reply:
Change made

Reply:
Still dont see this change?

Reply:
I made the change
its in my branch but it isnt updating the PR

Reply:
ok its there